account_invoice_line_report/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===========================
+Account Invoice Line Report
+===========================
+
+This module creates a new view to manage invoice lines information.
+
+Configuration
+=============
+
+In order to see invoice line information, you must install sales management
+app.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Accounting > Reporting > Business Intelligence > Invoice Lines
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/94/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/account-invoice-reporting/issues>`_. In case of trouble
+, please check there if your issue has already been reported. If you spotted
+it first, help us smashing it by providing a detailed and welcomed feedback.
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
```

Comment:
You should add the original author:

    Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>

Reply:
You should also add yourself